src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/RegexGeneratorHelper.netcoreapp.cs

```diff
@@ -23,7 +23,7 @@ namespace System.Text.RegularExpressions.Tests
 {
     public static class RegexGeneratorHelper
     {
-        private static readonly CSharpParseOptions s_previewParseOptions = CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview);
+        private static readonly CSharpParseOptions s_previewParseOptions = CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.Preview).WithDocumentationMode(DocumentationMode.Diagnose);
```

Comment:
>WithDocumentationMode [](http://example.com/codeflow?start=147&length=21)

Is this so we detect errors whenever we are emitting comments on the code?

Reply:
Yup. I'd had a bug where I was missing a closing bracket causing use in VS to fail but tests were still passing.